addons/spectrum/functions/fnc_spectrumEnableSettingChanged.sqf

```diff
@@ -29,7 +29,7 @@ if (_value) then {
 	GVAR(LastSpectrumMuzzleAttachment) = "";
 
 	// due to best practices we are gonna put the track loop in unscheduled space. 
-	GVAR(PFH_beaconPlayer) = [FUNC(spectrumTrackingLocal), 0.2] call CBA_fnc_addPerFrameHandler; 
+	GVAR(PFH_beaconPlayer) = [FUNC(spectrumTrackingLocal), 0.1] call CBA_fnc_addPerFrameHandler; 
```

Comment:
I also tried it with `0.1` but on my machine I did not get a significantly nicer sweep display. 
`0.2` looked almost the same and was easier on my FPS, so I stuck with it.

Does your setup look nicer with `0.1`?

Reply:
Oops... my bad. 😊 
I thought you suggested 0.1. 

I changed the repetition interval because the original interval makes the sweep look ugly (to few peaks shown before looping back or to slow a frequency increment when insisting on more peaks).

`0.1` was the best tradeoff between beauty and FPS that I could find.

This PFH parameter and `_sweepIncrementSpeed` in `fnc_getNextSweepFreq.sqf` can be somewhat balanced against each other if you want to tinker with those.